fronts-client/src/declarations/modules.d.ts

```diff
@@ -130,3 +130,8 @@ declare module 'react-router-redux' {
 		path: string,
 	): (state: { router: RouterState }) => match | null;
 }
+
+declare module 'history' {
```

Comment:
I was getting a typescript compile error on `yarn build` that there was no types for the 'history' package - added this stub to resolve that